firefox/manifest.json

```diff
@@ -23,6 +23,7 @@
 	},
 	"background": {
 		"scripts": [
+			"{{./firefox.entry.js}}",
```

Comment:
Technically this doesn't need to be on the background, but its tiny, and this is consistent with `edge.entry.js`